recipe/meta.yaml

```diff
@@ -2,20 +2,26 @@ package:
     name: mingwpy
     version: 0.1.0
 
-source:                                                                                                # [win]
+source:
```

Comment:
So, this line was not random. We added it because it broke the automation script for updating teams and the feedstocks repo. See this [line]( https://travis-ci.org/conda-forge/conda-forge.github.io/jobs/132453176#L550 ).

I'm going to add it back so this runs again. Please leave it in the future even if it is a mild irritance.